### PR TITLE
Create screenshot_f3.json

### PR DESCRIPTION
**Pull request by nhblock**

### CODE REVIEW

File: docs/json/screenshot_f3.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
+{
+  "title": “Maps F3 to Clipboard Screenshot”,
```
Comment (LinusU) [inline docs/json/screenshot_f3.json]: These quotes aren't the correct ones, it needs to be this one -> `"`

File: docs/json/screenshot_f3.json [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+  "rules": [
+    {
+      "description": "F3 to Clipboard Screenshot",
+      "manipulators": [
+        {
+          "type": "basic",
+          "from": {
+            “key_code”: “f3”
```
Comment (LinusU) [inline docs/json/screenshot_f3.json]: same here

**Comment by bingocaller** [comment]
I can't see a difference between this PR and #477 (except the Smart Quotes issue). @nhblock would you mind closing either this or #477?

File: docs/json/screenshot_f3.json (hunk 1 above)
Comment (nhblock) [reply docs/json/screenshot_f3.json]: Updated, thanks.

**Comment by nhblock** [comment]
@bingocaller closed #477, thank you.

**Comment by tekezo** [comment]
Thanks!

**Pull request merged**